packages/devtools_app/lib/src/error_badge_manager.dart

```diff
@@ -65,15 +102,43 @@ class ErrorBadgeManager extends DisposableController
     notifier.value = currentCount + 1;
   }
 
+  void appendInspectorErroredWidget(InspectableWidgetError error) {
+    var currentErrors = _activeErroredWidgets.value;
+    if (!currentErrors.any((e) => e.inspectorRef == error.inspectorRef)) {
+      currentErrors = [...currentErrors, error];
```

Comment:
would it be better to use a set instead of a list?

Reply:
To avoid the `any` enumerating through it, or something else? I think using a set would also require enumerating (since we're checking for the same ID rather than full same object), and both require creating a new object because the notifier only fires if `.value` changes.

Changing it to a `Map<String, DevToolsError>` where the ID is also the key makes this easier (using `containsKey`) so I've done that (in 6f779c11ea3b3f9617548fed1b7e3b4e8f51982b), but lmk if this doesn't seem like a good change.

Reply:
I believe InspectorRef should work as a key type in a Map and that way it is clearer what the types are.